adapter/memquota2/config/config.proto

```diff
@@ -26,6 +26,7 @@ option (gogoproto.gostring_all) = false;
 
 message Params {
 	message Quota {
+		option (gogoproto.goproto_getters) = true;
```

Comment:
Need getters so we can abstract over `Quota` and `Override` message.
They both implement 
```
interface {
 GetValidDuration()
  GetMaxAmount()
}
```